org.freeciv.sdl2.yml

```
@@ -0,0 +1,53 @@
+app-id: org.freeciv.sdl2
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: freeciv-sdl2
+rename-icon: freeciv-client
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --filesystem=~/.freeciv:create
+modules:
+  - name: sdl2-gfx
+    build-options:
+      - arch:
+          - aarch64:
+              - config-opts:
+                  - --disable-mmx
```

```suggestion
    build-options:
      arch:
        aarch64:
          config-opts:
            - --disable-mmx
```

This is probably what's expected.

---

I looked SDL2_gfx usage a bit more closely, and for freeciv's case enable/disable mmx makes no difference. So we can simply disable it unconditionally, as done in latest version.